sourcegraph/about
# Build vs buy blog post

Adds a blogpost & along with image asset, closing #5564

### Changelog
- Adds new blog post `.md` along with the associated asset

### Test

1. Ensure prettier has standardized the proposed changes.
2. Navigate to `/blog/things-to-know-before-building-a-code-search-tool` and ensure blog post renders as expected
3. Test responsive sizing
4. Navigate to `/blog` index & ensure new blog card renders as expected
